RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -23,7 +23,10 @@ boolean LM_edTheUndying();						//Defined in autoscend/auto_edTheUndying.ash
 boolean LX_desertAlternate();
 boolean LX_hippyBoatman();
 boolean LX_lockPicking();
-boolean LX_phatLootToken();
+float estimateDailyDungeonAdvNeeded();				//Defined in autoscend/quests/level_any.ash
+boolean LX_phatLootToken();							//Defined in autoscend/quests/level_any.ash
+boolean LX_dailyDungeonToken();						//Defined in autoscend/quests/level_any.ash
+void dailyDungeonChoiceHandler(int choice, string[int] options);	//Defined in autoscend/quests/level_any.ash
```

Comment:
could you move this to the end of the file with the other choice handlers please?

Reply:
change made